app/templates/icon-catalog/index.hbs

```diff
@@ -17,7 +17,7 @@
         {{! Copied from the AuDataTable::TextSearch component since that doesn't support calling actions on input}}
         <div class="au-c-data-table__search">
           <AuInput
-            @value={{this.code}}
+            @value={{this.label}}
```

Comment:
I updated Appuniversum on the dev branch, Appuniversum v3 no longer supports `@value` so it should just be an attribute. I would rebase on dev before merging this.

```suggestion
            value={{this.label}}
```